Signal/src/call/NonCallKitCallUIAdaptee.swift

```diff
@@ -27,12 +27,10 @@ class NonCallKitCallUIAdaptee: CallUIAdaptee {
 
         let call = SignalCall.outgoingCall(localId: UUID(), remotePhoneNumber: handle)
 
-        CallService.signalingQueue.async {
-            _ = self.callService.handleOutgoingCall(call).then {
-                Logger.debug("\(self.TAG) handleOutgoingCall succeeded")
+        self.callService.handleOutgoingCall(call).then {
```

Comment:
Nope, I'm not seeing any such warning?  That's really a possible warning in swift?  It seems like there would be mostly false positives for such a check.